package.json

```diff
@@ -1,95 +1,95 @@
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "stories-check": "rimraf ./.stories-pending && storycap --serverCmd \"start-storybook --ci -p 6007\" --serverTimeout 180000 -o .stories-pending http://localhost:6007 && kill-port 6007 && node -e \"require('./helpers/ci/StoriesCompare').run()\""
-  },
-  "dependencies": {
-    "@emotion/cache": "^11.7.1",
-    "@emotion/react": "^11.9.0",
-    "@emotion/server": "^11.4.0",
-    "@emotion/styled": "^11.8.1",
-    "@mui/icons-material": "^5.6.2",
-    "@mui/material": "^5.6.3",
-    "@mui/styles": "^5.8.0",
-    "axios": "^0.26.1",
-    "clsx": "^1.1.1",
-    "mobx": "^6.6.0",
-    "mobx-react-lite": "^3.3.0",
-    "next": "^12.1.6",
-    "next-react-svg": "^1.1.1",
-    "noty": "^3.2.0-beta",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "sass": "^1.50.1",
-    "typescript": "^4.7.2",
-    "webpack": "^5.70.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.18.0",
-    "@babel/eslint-parser": "^7.17.0",
-    "@babel/plugin-proposal-decorators": "^7.18.2",
-    "@babel/preset-react": "^7.16.7",
-    "@cypress/react": "^5.12.5",
-    "@cypress/webpack-dev-server": "^1.8.4",
-    "@next/bundle-analyzer": "^12.0.10",
-    "@next/eslint-plugin-next": "^12.1.5",
-    "@storybook/addon-actions": "^6.4.22",
-    "@storybook/addon-essentials": "^6.4.22",
-    "@storybook/addon-links": "^6.4.22",
-    "@storybook/addons": "^6.4.22",
-    "@storybook/builder-webpack5": "^6.4.22",
-    "@storybook/manager-webpack5": "^6.4.22",
-    "@storybook/react": "^6.4.22",
-    "@zeit/next-source-maps": "^0.0.3",
-    "babel-loader": "^8.2.3",
-    "chalk": "4.1.2",
-    "cross-env": "^7.0.2",
-    "cypress": "^9.7.0",
-    "eslint": "^8.9.0",
-    "eslint-config-airbnb": "19.0.4",
-    "eslint-plugin-import": "2.25.4",
-    "eslint-plugin-jsx-a11y": "6.5.1",
-    "eslint-plugin-react": "7.29.4",
-    "eslint-plugin-react-hooks": "4.3.0",
-    "eslint-plugin-storybook": "^0.5.7",
-    "fs-extra": "^10.0.1",
-    "html-webpack-plugin": "5",
-    "jest": "^28.0.1",
-    "jest-environment-jsdom": "^28.0.1",
-    "kill-port": "^2.0.0",
-    "next-compose-plugins": "^2.2.0",
-    "postcss": "^8.4.14",
-    "puppeteer": "^13.5.1",
-    "rimraf": "^3.0.2",
-    "sass-loader": "^12.4.0",
-    "start-server-and-test": "^1.11.0",
-    "storybook-addon-next": "^1.6.2",
-    "storycap": "^3.1.7",
-    "webpack-dev-server": "4"
-  }
-}
+    "name": "react-starter",
+    "version": "0.1.0",
+    "private": true,
+    "scripts": {
+        "dev": "next dev",
+        "dev-local": "cross-env NEXT_PUBLIC_LOCAL_SERVER=true next dev",
+        "build": "next build",
+        "build-local": "cross-env NEXT_PUBLIC_LOCAL_SERVER=true next build",
+        "build-analyze": "cross-env ANALYZE=true yarn build",
+        "export": "rimraf ./out && rimraf ./.next && cross-env NODE_ENV=production next build && next export",
+        "start": "next start",
+        "lint": "next lint",
```

Comment:
@AleksandarDev what's up with `lint-fix`?

Reply:
You can use `yarn lint --fix`

Reply:
Hmm okay